src/client/components/TestComponent/Reservations.js

```diff
@@ -1,24 +1,37 @@
 import React from "react";
-import {useState,useEffect} from "react"
+import { useState, useEffect } from "react";
 
-export default function Reservations(){
-const [phoneNumber,setPhoneNumber]=useState("");
-const [name,setName]=useState("");
-const [email,setEmail]=useState("");
-
-
-
-
-return(
+export default function Reservations() {
```

Comment:
Is this component used somewhere? If not, better to delete it.
Also TestComponent isn't a very meaningful name :)